front/src/reducers/osrdconf/types.ts

```diff
@@ -91,6 +91,25 @@ export type PathStep = PathItemLocation & {
   isInvalid?: boolean;
 };
 
-export type StdcmPathStep = PathStep & {
-  tolerances?: { before: number; after: number };
-} & ({ isVia: true; stopType: StdcmStopTypes } | { isVia: false; arrivalType: ArrivalTimeTypes });
+export type UicPathItemLocation = {
+  uic: number;
+  secondary_code?: string | null;
+  name: string;
+  coordinates: [number, number];
+};
```

Comment:
Maybe we could find another name because a PathItemLocation in the editoast way doesn't have a name or coordinates, it might be misleading.

Also, the type could be more dynamic using this syntax : 

```suggestion
type SomeCoolName = Extract<PathItemLocation, { uic: number }> & {
  name: string;
  coordinates: [number, number];
};
```

We don't have a use for other cases for now but we could also do 
```suggestion
type SomeCoolName<T> = Extract<PathItemLocation, T> & {
  name: string;
  coordinates: [number, number];
};
```

Reply:
You're right, it's done :)